config/storage.yml

```diff
@@ -1,7 +1,3 @@
 local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
-
-production:
-  service: Disk
-  root: /home/deploy/lobsters/shared/shared/storage
```

Comment:
If `Rails.root.join("storage")` symlinks to `/home/deploy/lobsters/shared/shared/storage`, there is no need to define another Disk service for production :thinking:

Reply:
Ah, yeah, I found the documentation at https://api.rubyonrails.org/v8.0.0/classes/ActiveStorage/Service.html and I see that these two sections are equivalent.